Source/Core/CameraLimiter.js

```diff
@@ -0,0 +1,74 @@
+define([
+    '../Core/Cartesian3',
+    '../Core/Check',
+    '../Core/defaultValue',
+    '../Core/defined',
+    '../Core/HeadingPitchRoll',
+    '../Core/Math'
+], function(
+    Cartesian3,
+    Check,
+    defaultValue,
+    defined,
+    HeadingPitchRoll,
+    CesiumMath) {
+    'use strict';
+
+    /**
+     * Object used to define bounds limiting the {@link @Camera#position} when in 3D mode.
+     *
+     * {@link @Camera#cameraLimiter}
+     *
+     * If set, the camera of this limiter will be constrained by location.
```

Comment:
If what is set? Maybe `used by a {@link Camera} to limit the location of it's movement`?